pkg/ast/ast_operation_definition.go

```diff
@@ -16,6 +17,20 @@ const (
 	OperationTypeSubscription
 )
 
+func (t OperationType) Name() string {
+	switch t {
+	case OperationTypeUnknown:
+		return "unknown"
+	case OperationTypeQuery:
+		return "query"
+	case OperationTypeMutation:
+		return "mutation"
+	case OperationTypeSubscription:
+		return "subscription"
+	}
+	return fmt.Sprintf("unknown operation type %d", int(t))
```

Comment:
I'd expect an empty string or panic.

Reply:
A `panic()` seemed a bit too harsh, but I have no strong feelings about it. I've changed it to panic, PTAL